### PR TITLE
A couple of minor documentation clarifications, for #6026 & #6038.

### PR DESCRIPTION
**Pull request by jehturner**
Document limitations discussed in #6026 & #6038, until they're resolved.

### CODE REVIEW

File: docs/modeling/fitting.rst [hunk 1]
```diff
@@ -21,6 +21,14 @@ The rules for passing input to fitters are:
   argument just as used when evaluating models; this may be required for the
   fitter to know how to broadcast the input data.
 
+* The `~astropy.modeling.fitting.LinearLSQFitter` currently works only with
+  simple (not compound) models.
+
+* The current fitters work only with models that have a single output
+  (including bivariate functions such as
+  `~astropy.modeling.polynomial.Chebyshev2D` but not compound models that map
+  x, y -> x', y').
```
Comment (pllim) [inline docs/modeling/fitting.rst]: This might need to be encapsulated within double ticks. Try to build the doc locally and see if it looks nice.

**Comment by jehturner** [comment]
It looks OK to me (depending on whether you find the bold blue links to LinearLSQFitter and Chebyshev2D distracting).

**Comment by pllim** [comment]
No, I meant
```
x, y -> x', y'
```

**Comment by pllim** [comment]
Maybe attach a screenshot of the rendered doc?

**Comment by jehturner** [comment]
fitting_models_screenshot

**Comment by jehturner** [comment]
The alternative (depending what you put in backticks) looks like this:

fitting_models_screenshot_2

**Comment by pllim** [comment]
I prefer the alternative, but @nden should approve the actual wordings before merge.

**Comment by nden** [comment]
Looks good. Thanks @jehturner.

**Pull request merged**